docs/events/api/timeline-api-update.asciidoc

```diff
@@ -0,0 +1,108 @@
+[[timeline-api-update]]
+=== Add a note to existing timeline
```

Comment:
Might want to add "an" before "Timeline" in the topic title. Also recommend capitalizing the "t" in "Timeline".